scripts/deploy.sh

```diff
@@ -34,11 +34,74 @@ function upload_assets {
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-es.pdf" "${SITE_HOME}/assets/tldr-book-es.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-fa.pdf" "${SITE_HOME}/assets/tldr-book-fa.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-fr.pdf" "${SITE_HOME}/assets/tldr-book-fr.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-hi.pdf" "${SITE_HOME}/assets/tldr-book-hi.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-id.pdf" "${SITE_HOME}/assets/tldr-book-id.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-it.pdf" "${SITE_HOME}/assets/tldr-book-it.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-lo.pdf" "${SITE_HOME}/assets/tldr-book-lo.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-ml.pdf" "${SITE_HOME}/assets/tldr-book-ml.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-ne.pdf" "${SITE_HOME}/assets/tldr-book-ne.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-nl.pdf" "${SITE_HOME}/assets/tldr-book-nl.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-no.pdf" "${SITE_HOME}/assets/tldr-book-no.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-pl.pdf" "${SITE_HOME}/assets/tldr-book-pl.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-pt_BR.pdf" "${SITE_HOME}/assets/tldr-book-pt_BR.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-pt_PT.pdf" "${SITE_HOME}/assets/tldr-book-pt_PT.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-ro.pdf" "${SITE_HOME}/assets/tldr-book-ro.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-ru.pdf" "${SITE_HOME}/assets/tldr-book-ru.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-sh.pdf" "${SITE_HOME}/assets/tldr-book-sh.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-sr.pdf" "${SITE_HOME}/assets/tldr-book-sr.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-sv.pdf" "${SITE_HOME}/assets/tldr-book-sv.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-ta.pdf" "${SITE_HOME}/assets/tldr-book-ta.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-th.pdf" "${SITE_HOME}/assets/tldr-book-th.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-tr.pdf" "${SITE_HOME}/assets/tldr-book-tr.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-uk.pdf" "${SITE_HOME}/assets/tldr-book-uk.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-uz.pdf" "${SITE_HOME}/assets/tldr-book-uz.pdf"
+
 
   sha256sum \
     "${SITE_HOME}/assets/index.json" \
     "${SITE_HOME}/assets/"*.zip \
     "${SITE_HOME}/assets/tldr-book.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ar.pdf" \
+    "${SITE_HOME}/assets/tldr-book-bn.pdf" \
+    "${SITE_HOME}/assets/tldr-book-bs.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ca.pdf" \
+    "${SITE_HOME}/assets/tldr-book-da.pdf" \
+    "${SITE_HOME}/assets/tldr-book-de.pdf" \
+    "${SITE_HOME}/assets/tldr-book-es.pdf" \
+    "${SITE_HOME}/assets/tldr-book-fa.pdf" \
+    "${SITE_HOME}/assets/tldr-book-fr.pdf" \
+    "${SITE_HOME}/assets/tldr-book-hi.pdf" \
+    "${SITE_HOME}/assets/tldr-book-id.pdf" \
+    "${SITE_HOME}/assets/tldr-book-it.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ja.pdf" \
```

Comment:
don't think this file exists?

Reply:
Yeah, you are right, I accidentally specified it here. Will update it to wildcard instead as you suggested.